packages/app_review/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ author: Rody Davis <rody.davis.jr@gmail.com>
 homepage: https://github.com/AppleEducate/plugins
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
```

Comment:
FWIW -- I don't think this is the right change.. you are going back to 2.0.0 and it should be a 2.1.0+ or something... 

I think you mean:

sdk: '>=2.1.0-dev.68.0 <3.0.0'

Reply:
This version is directly from the example project Android Studio creates and is the version that most other compatible plugins use. >=2.0.0-dev.68.0 <3.0.0 simply means an SDK between those two values, and not necessarily going back to 2.0.0.  It's better for backwards compatibility and allows for this plugin's usage across more SDKs

Reply:
OK -- as long as it works -- but I think many many plugins now demand SDK >= 2.1.XX and 2.0 will be flagged as an error if they stay there.. <shrug>

Reply:
I originally had it for 2.1.0 but changed it as there are a lot of plugins that haven't updated yet and some of my plugins depend on other plugins. Once they update I will too.